app/application/services/auth/jwt_service.py

```
@@ -0,0 +1,19 @@
+from app.core.security import JWTProvider
+
+
+class JWTService:
+    def __init__(self, provider: JWTProvider = None):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)